presto-hive/src/main/java/io/prestosql/plugin/hive/ViewReaderUtil.java

```diff
@@ -97,6 +102,13 @@ public static String encodeViewData(ConnectorViewDefinition definition)
         return VIEW_PREFIX + data + VIEW_SUFFIX;
     }
 
+    public static String encodeMaterializedViewData(ConnectorMaterializedViewDefinition definition)
```

Comment:
This method doesn't belong in this commit.

Reply:
Moving to iceberg connector commit